evm_loader/utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Permissions management scripts
+
+Scripts mint_permission_token.sh mint_many.sh are suited to operate with client's and contract's permission token balances - to grant or deprive permissions.
+
+## Requirements
+
+To use this scripts one should have next CLI utilities installed:
+    - solana CLI interface
+    - neon-cli 
+    - spl-token program
+
+Non-zero SOL balance is needed on selected Solana network on default account ~/.config/solana/id.json
+Also, user should have correct Mint-authority JSON file
+
+## Permission management for a single client/contract
+
+mint_permission_token.sh operates with permission tokens balances of a signle client/contract. Script will estimate current state of client's/contract's permission tokens balances and mint selected tokens if needed. Here is common form of calling this script:
```

Comment:
I wouldn't use a 'blockchain' term for logically non-blockchain thing: if we set allow/deny permissions, names of scripts could be specific, like, for example:
```
set_single_permission.sh
set_multiple_permissions.sh
```

Reply:
Files renamed